### ISSUE TITLE
File not found at path: vuepress/.assets

### ISSUE THREAD

**Issue by dimaslanjaka** [nelson6e65/phpdoc-vuepress#33]
**Scopes**

- [x] Installation
- [ ] Documentation
- [ ] Template (twig, xml)
- [ ] PHP sources
- [ ] Output HTML
- [ ] Output Markdown
- [ ] Appearance
- [ ] Other

**Describe the bug**
```bash
Parsing files

Applying transformations (can take a while)
21:22:43 ERROR     [console] Error thrown while running command "-d="src/" -t="docs/php/" --template="libs/src/phpdoc-vuepress/data/templates/vuepress". Message: "File not found at path: vuepress/.assets" ["exception" => League\Flysystem\FileNotFoundException^ { …},"command" => "-d="src/" -t="docs/php/" --template="libs/src/phpdoc-vuepress/data/templates/vuepress"","message" => "File not found at path: vuepress/.assets"]

In Filesystem.php line 389:

  File not found at path: vuepress/.assets  
```

**Steps to reproduce**
```bash
phpdoc -d="src/" -t="docs/php/" --template="libs/src/phpdoc-vuepress/data/templates/vuepress"
```

**Comment by nelson6e65** [comment]
Hi. Thanks for reporting.

Are you using a phar version of phpdoc?

What version are you using?

**Comment by dimaslanjaka** [comment]
Yes. i used phpdoc 3.0.0.

**Comment by dimaslanjaka** [comment]
[https://github.com/dimaslanjaka/universal-framework/tree/master/libs/bin/phpdoc](https://github.com/dimaslanjaka/universal-framework/tree/master/libs/bin/phpdoc)

this is my phpdoc

**Comment by dimaslanjaka** [comment]
phpdocblock version ^2.9.1 as described on [Doc](https://nelson6e65.github.io/phpdoc-vuepress/guide/getting-started.html#requirements) not working as well.

![image](https://user-images.githubusercontent.com/12471057/118732938-8ce5a500-b865-11eb-9aeb-081e092f0801.png)

**Comment by nelson6e65** [comment]
I think you are using phpDocumentor 3. Sadly, at the moment, this template is not compatible with it.

Seems that you installed locally with composer, but still using phpDocumentor 3 executable. Try to run `vendor/bin/phpdoc` directly instead of simply `phpdoc`.

And yes, PHAR bins seems to fail to locate template files also. :(

You can try to build the documentation using GitHub Actions, for example the Build job in this workflow: https://github.com/nelson6e65/php_nml/blob/master/.github/workflows/documentation.yml#L47 (Is configured to use Netlify, but you can adapt it to GitHub pages or any other).


If you like to discard if there are some problems with your config, you can try to build the documentation of this repository (you will need PHP 7.2 or 7.3 in your environment):

```sh
# Clone and move to phpdoc-vuepress directory

# Install dependencies
composer install
npm install

# Build API with local phpDoc configured as composer script
composer build:api

# Run VuePress server
npm run docs:dev
```

**Comment by nelson6e65** [comment]
If you have problems with building docs for this repository, following previous instructions, then there is some platform incompatibility (PHP version, mostly).

If you successfully build the project, then there are some issues with the global binary files you are using. Maybe duplicated phpdoc bin in different locations.

**Comment by nelson6e65** [comment]
You can try using composer scripts, [like this repo](https://github.com/nelson6e65/phpdoc-vuepress/blob/master/composer.json#L39), if you installed phpDocumentor 2.9.1 in your local composer.json. With this, composer will look for local install at first.

**Comment by dimaslanjaka** [comment]
Now render running properly, but exceeds 500kb ?

![image](https://user-images.githubusercontent.com/12471057/118847388-e26a9200-b8f7-11eb-9673-b6f5d66b7f9f.png)

**Comment by nelson6e65** [comment]
What do you mean with "exceeds 500Kb"?

**Comment by dimaslanjaka** [comment]
> What do you mean with "exceeds 500Kb"?

i dont know, its mean too large md file.

**Comment by dimaslanjaka** [comment]
are this no alternative splitting classes.md when reached 500kb size ?
seems like, splitting other php to classes1.md when classes.md reach 500kb and so on.

**Comment by dimaslanjaka** [comment]
when classes.md more than 500kb, vuepress cannot load all contents. result is blank on classes.md

**Comment by dimaslanjaka** [comment]
Config : [https://github.com/dimaslanjaka/universal-framework/tree/master/docs-src](https://github.com/dimaslanjaka/universal-framework/tree/master/docs-src)
Github Page : [http://git.webmanajemen.com/universal-framework/php/](http://git.webmanajemen.com/universal-framework/php/)

**Comment by dimaslanjaka** [comment]
![image](https://user-images.githubusercontent.com/12471057/119059055-bf280b80-b9f9-11eb-9f04-aee7b01095b5.png)

now working like a charm. but i loses a lot function description and usages :(

**Comment by nelson6e65** [comment]
> @dimaslanjaka 
> when classes.md more than 500kb, vuepress cannot load all contents. result is blank on classes.md (http://localhost:8080/base-url/classes.html)

Oh, I understand. You have too many classes, so the file is too large. Hmm... 


Yes, I was thinking a way to split on multiple files some time ago, and I found some directions about it, but for my implementation for **phpDocumentor 3**. 

At the moment I have not much time to invest in here. I'm sorry 😥. I will try to get more tome to invest in that implementation in the following weeks, because I also need PHP 7.4+ syntax compatibility for my current projects' documentation.

**Comment by dimaslanjaka** [comment]
I have idea.
Create single file php to splitting the Larger markdowns.
Then creating json config to output splitted md's.
And referring json file to config.js.

I want to help, but i still confusing on *how md marker to indicate next page or next definition class/method*. Need split them to spread into multiple files

**Comment by dimaslanjaka** [comment]
Executing that single file php on cli faster than webserver. Can load 3mb++ less than 3 seconds on CLI.

**Comment by nelson6e65** [comment]
Yes, the spitting was a challenge when I try to implement, but I noticed a feature in phpDocumentor 3 to allow split documentation in multiple files, `1 php  --> 1 md`, but the problem was linking files on VuePress menu and also linking between files on different files and type (global function, class, trait, interface, ...).

At the beginning, my intention was to build only 1 JSON file and start building from there. 1 Generic Vue component for Class, Trait, etc, but I was kind of difficult, and I ended up by using twig templates instead.

**Comment by dimaslanjaka** [comment]
[https://github.com/fr3nch13/phpdoc-markdown](https://github.com/fr3nch13/phpdoc-markdown) i have try this.
good result, but i dont know how to port it into vuepress
